src/services/socketIoServices.ts

```diff
@@ -0,0 +1,169 @@
+'use client'
```

Comment:
nit: This service you could simply call it "socket" or "api", no need to say "socketServices" ☺️